### ISSUE TITLE
Can't install perspective-python on python3.9, macOS Big Sur

### ISSUE THREAD

**Issue by jmy48** [finos/perspective#1603]
Failed building wheel:
```
  running build_ext
  Traceback (most recent call last):
    File "/Users/joshuaying/_/ida/raid_env/lib/python3.9/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 363, in <module>
      main()
    File "/Users/joshuaying/_/ida/raid_env/lib/python3.9/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 345, in main
      json_out['return_val'] = hook(**hook_input['kwargs'])
    File "/Users/joshuaying/_/ida/raid_env/lib/python3.9/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 261, in build_wheel
      return _build_backend().build_wheel(wheel_directory, config_settings,
    File "/private/var/folders/r0/sbnbrfln46jcm0dfxchjzmww00tlq6/T/pip-build-env-tj_zw5gx/overlay/lib/python3.9/site-packages/setuptools/build_meta.py", line 230, in build_wheel
      return self._build_with_temp_dir(['bdist_wheel'], '.whl',
    File "/private/var/folders/r0/sbnbrfln46jcm0dfxchjzmww00tlq6/T/pip-build-env-tj_zw5gx/overlay/lib/python3.9/site-packages/setuptools/build_meta.py", line 215, in _build_with_temp_dir
      self.run_setup()
    File "/private/var/folders/r0/sbnbrfln46jcm0dfxchjzmww00tlq6/T/pip-build-env-tj_zw5gx/overlay/lib/python3.9/site-packages/setuptools/build_meta.py", line 267, in run_setup
      super(_BuildMetaLegacyBackend,
    File "/private/var/folders/r0/sbnbrfln46jcm0dfxchjzmww00tlq6/T/pip-build-env-tj_zw5gx/overlay/lib/python3.9/site-packages/setuptools/build_meta.py", line 158, in run_setup
      exec(compile(code, __file__, 'exec'), locals())
    File "setup.py", line 222, in <module>
      setup(
    File "/private/var/folders/r0/sbnbrfln46jcm0dfxchjzmww00tlq6/T/pip-build-env-tj_zw5gx/overlay/lib/python3.9/site-packages/setuptools/__init__.py", line 159, in setup
      return distutils.core.setup(**attrs)
    File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/distutils/core.py", line 148, in setup
      dist.run_commands()
    File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/distutils/dist.py", line 966, in run_commands
      self.run_command(cmd)
    File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/distutils/dist.py", line 985, in run_command
      cmd_obj.run()
    File "/private/var/folders/r0/sbnbrfln46jcm0dfxchjzmww00tlq6/T/pip-build-env-tj_zw5gx/overlay/lib/python3.9/site-packages/wheel/bdist_wheel.py", line 299, in run
      self.run_command('build')
    File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/distutils/cmd.py", line 313, in run_command
      self.distribution.run_command(command)
    File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/distutils/dist.py", line 985, in run_command
      cmd_obj.run()
    File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/distutils/command/build.py", line 135, in run
      self.run_command(cmd_name)
    File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/distutils/cmd.py", line 313, in run_command
      self.distribution.run_command(command)
    File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/distutils/dist.py", line 985, in run_command
      cmd_obj.run()
    File "setup.py", line 91, in run
      self.run_cmake()
    File "setup.py", line 96, in run_cmake
      out = subprocess.check_output([self.cmake_cmd, "--version"])
    File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/subprocess.py", line 424, in check_output
      return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
    File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/subprocess.py", line 505, in run
      with Popen(*popenargs, **kwargs) as process:
    File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/subprocess.py", line 951, in __init__
      self._execute_child(args, executable, preexec_fn, close_fds,
    File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/subprocess.py", line 1696, in _execute_child
      and os.path.dirname(executable)
    File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/posixpath.py", line 152, in dirname
      p = os.fspath(p)
  TypeError: expected str, bytes or os.PathLike object, not NoneType
  ----------------------------------------
  ERROR: Failed building wheel for perspective-python
Failed to build perspective-python
ERROR: Could not build wheels for perspective-python, which is required to install pyproject.toml-based projects


```

**Comment by texodus** [comment]
Hi @jmy48 thanks for the report!

Apologies for this, I had not yet uploaded the wheels for this build yet due to some failures we discovered in our automated deployment branch, only the sdist which should mostly only be necessary as a backup.

I've uploaded the `1.0.1` wheels for OSX/Python3.9 (and others) just now, try uninstalling and installing again and it should pick up the pre-built wheels - let me know if this doesn't work.

If you're interested in installing perspective from sdist in the future for whatever reason, there is more information [in the dev docs](https://perspective.finos.org/docs/md/development.html).
